x-pack/plugins/cases/public/methods/get_all_cases.tsx

```diff
@@ -8,10 +8,14 @@
 import { EuiLoadingSpinner } from '@elastic/eui';
 import React, { lazy, Suspense } from 'react';
 import { AllCasesProps } from '../components/all_cases';
+import { OwnerProvider } from '../components/owner_context';
+import { Owner } from '../types';
 
 const AllCasesLazy = lazy(() => import('../components/all_cases'));
-export const getAllCasesLazy = (props: AllCasesProps) => (
-  <Suspense fallback={<EuiLoadingSpinner />}>
-    <AllCasesLazy {...props} />
-  </Suspense>
+export const getAllCasesLazy = (props: AllCasesProps & Owner) => (
+  <OwnerProvider owner={props.owner}>
```

Comment:
@michaelolo24 Should be moved inside `AllCasesLazy` component?

Reply:
Ah, you can ignore my other question about why the props a typed as `ComponentProps & Owner`. If you nove them into the components then you can combine the prop types to just `ComponentProps`. It's really up to you 😄. It's not hurting performance or anything either way, might just be cleaner / easier to reason about when moved in to the component. Whatever you decide to do, just be consistent across all the components 🤷🏾 

As an aside, I'm not really sure if the provider is affected by being outside of the `<Suspense>`? Doubtful, but maybe worth double checking